Sources/StreamChatUI/UIConfig+ChannelList.swift

```diff
@@ -25,14 +25,14 @@ public extension _UIConfig {
         /// The Cell Separator View.
         public var cellSeparatorReusableView: UICollectionReusableView.Type = _CellSeparatorReusableView<ExtraData>.self
 
-        /// The base view of the channel list item to support swipeable gestures.
-        public var swipeableItemView: _ChatChannelSwipeableListItemView<ExtraData>.Type =
-            _ChatChannelSwipeableListItemView<ExtraData>.self
+        /// `SwibeableView` instance wrapped in the cell to support action views on swipe in the cell.
+        public var swipeableView: _SwipeableView<ExtraData>.Type =
+            _SwipeableView<ExtraData>.self
```

Comment:
Yes! This is the style of documentation we need 🎉

Reply:
🎉